main.py

```diff
@@ -44,7 +43,10 @@ def __init__(self):
 
     def showMapScreen(self):
         self.stack.setCurrentIndex(1)
-    
+    """
+    Enter fight screen by pointing stack at fight screen. 
+    This will need to be changed to set the proper opponent per setting. Index 2 is the fight screen.
+    """
     def EnterFight(self):
```

Comment:
The docstring has to go inside the function.
```suggestion
    def EnterFight(self):
     """
    Enter fight screen by pointing stack at fight screen. 
    This will need to be changed to set the proper opponent per setting. Index 2 is the fight screen.
    """
```